pytorch_lightning/callbacks/base.py

```diff
@@ -89,15 +89,15 @@ def on_validation_epoch_start(self, trainer, pl_module: LightningModule) -> None
         """Called when the val epoch begins."""
         pass
 
-    def on_validation_epoch_end(self, trainer, pl_module: LightningModule) -> None:
+    def on_validation_epoch_end(self, trainer, pl_module: LightningModule, outputs: Any) -> None:
         """Called when the val epoch ends."""
```

Comment:
Can you add `Args:` to the docstring? For all `on_*_epoch_end` methods

Reply:
I see you updated the pytorch_lightning/trainer/callback_hook.py docstrings but not these

Reply:
Well, why just `on_*_epoch_end` methods then? You can't just add Args for one parameter "outputs". You should add it for all arguments, also should we add them in all the methods in the Callback class? Because the methods have initial similar arguments.

Reply:
> You can't just add Args for one parameter "outputs". You should add it for all arguments

Yeah, I'm not saying we should only describe `outputs`

> should we add them in all the methods in the Callback class? Because the methods have initial similar arguments.

I'd say yes, but that's out of the scope of this PR

Reply:
Yup, will do a follow-up PR for types for all methods.